### PR TITLE
Tweaks to the language of the Node.JS document

### PR DESCRIPTION
**Pull request by Nooshu**
Minor language updates, hopefully the document now reads slightly better.

### CODE REVIEW

File: source/manuals/programming-languages/nodejs/index.html.md.erb [hunk 1]
```diff
@@ -12,26 +12,15 @@ section has more detail on how we currently use Node.js at GDS and why this
...
-lints your code. [Standard] will autofix small errors then continue with your 
-commit. If there’s something it can’t fix then the commit fails with a useful 
-error message such as:
+To maintain a consistent and readable JavaScript codebase we use [StandardJS]. This includes a built in formatter, linter and automatic code fixer (auto-fix). To make sure everything stays clean we recommend running this tool before each commit. It is possible to setup a pre-commit hook to automate this process. When [StandardJS] auto-fix's errors it has clear and concise messaging and it also tells you when an automatic fix isn't possible.
+
+We’ve encapsulated our recommended process into a separate [repository](https://github.com/jonheslop/nodejs-starter) where everything is controlled by a few lines of config within your `package.json` file. Once you have run `npm install`, any time you make a commit, [StandardJS] runs. It will then format and lint your code automatically. [StandardJS] will auto-fix obvious errors then continue with your commit. If there’s something it can’t fix, then the commit fails with a useful error message such as:
```
Comment (maxcbc) [inline source/manuals/programming-languages/nodejs/index.html.md.erb]: Should we not move jonheslop/nodejs-starter to alphagov

Comment (jonheslop) [reply source/manuals/programming-languages/nodejs/index.html.md.erb]: Yes definitely, but let’s not block this PR, I'll raise a separate issue for that. I can ask David/Dom/Will if they can create a repo for me.

File: source/manuals/programming-languages/nodejs/index.html.md.erb [hunk 2]
```diff
@@ -12,26 +12,15 @@ section has more detail on how we currently use Node.js at GDS and why this
...
-[Standard] can auto-fix common errors and have clear error messaging when auto-fix 
-isn’t possible. 
-
-We’ve encapsulated our recommended process into a [repo](https://github.com/jonheslop/nodejs-starter) where everything 
-is controlled by a few lines of config within your `package.json`. Once you have 
-run `npm install`, any time you make a commit [Standard] runs which formats and 
-lints your code. [Standard] will autofix small errors then continue with your 
-commit. If there’s something it can’t fix then the commit fails with a useful 
-error message such as:
+To maintain a consistent and readable JavaScript codebase we use [StandardJS]. This includes a built in formatter, linter and automatic code fixer (auto-fix). To make sure everything stays clean we recommend running this tool before each commit. It is possible to setup a pre-commit hook to automate this process. When [StandardJS] auto-fix's errors it has clear and concise messaging and it also tells you when an automatic fix isn't possible.
```
Comment (joelanman) [inline source/manuals/programming-languages/nodejs/index.html.md.erb]: `fix's` isn't grammatically correct - how about `automatically fixes`?

File: source/manuals/programming-languages/nodejs/index.html.md.erb (hunk 1 above)
Comment (maxcbc) [reply source/manuals/programming-languages/nodejs/index.html.md.erb]: Cool, lets get that done. Approved on that assumption

File: source/manuals/programming-languages/nodejs/index.html.md.erb (hunk 2 above)
Comment (Nooshu) [reply source/manuals/programming-languages/nodejs/index.html.md.erb]: Good spot, updated

**Pull request merged**